core/common/src/main/java/alluxio/util/ConfigurationUtils.java

```diff
@@ -512,24 +537,110 @@ public static AlluxioConfiguration loadClusterDefaults(InetSocketAddress address
           continue;
         }
         String value = property.getValue();
-        clusterProps.put(key, value);
-        LOG.debug("Loading cluster default: {} ({}) -> {}", key, key.getScope(), value);
+        props.put(key, value);
+        LOG.debug("Loading property: {} ({}) -> {}", key, key.getScope(), value);
```

Comment:
this is misleading since it doesn't specify the path, so it looks like we're loading a regular cluster-wide default

Reply:
updated with a better logging approach